src/main/java/taxi/exception/AuthenticationException.java

```diff
@@ -0,0 +1,8 @@
+package taxi.exception;
+
+public class AuthenticationException extends Exception {
+
+    public AuthenticationException(String message) {
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/230331435-c7963efb-c83c-409b-bd2d-5410aab4c452.png)

Reply:
Oh, sorry, I checked the checklist but completely forgot to check AuthenticationException.